packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -332,10 +370,18 @@ class TextSelectionOverlay {
     );
   }
 
+  /// {@template flutter.widgets.SelectionOverlay.context}
+  /// The context in which the selection UI should appear.
+  ///
+  /// This context must have an [Overlay] as an ancestor because this object
+  /// will display the text selection handles in that [Overlay].
+  /// {@endtemplate}
+  final BuildContext context;
+
   /// Controls the fade-in and fade-out animations for the toolbar and handles.
   @Deprecated(
     'Use `SelectionOverlay.fadeDuration` instead. '
-    'This feature was deprecated after v2.12.0-4.1.pre.'
+    'This feature was deprecated after v3.3.0-0.5.pre.'
```

Comment:
I remember this was deprecated a while back, why the version change?

Reply:
Whoops, this must have been my overzealous find and replace when I changed the deprecated version number in the deprecations that I made in this PR.